### DISCUSSION TITLE
Software Engineering Survey on GitHub Discussions.

### DISCUSSION THREAD

**Discussion by raux** [rabbitmq/rabbitmq-server#2429] [Other] [answered]
Do you find GitHub discussions useful as a communication channel? Maybe not? 
Hello Developers! I am Raula Gaikovina Kula, a Software Engineering researcher
https://raux.github.io/

Together with a team of researchers, we are interested in your thoughts about
using the GitHub Discussions Beta. We noticed that your project is currently using.

We have developed a simple 3 question survey that could take up to 5 mins to complete.
https://forms.gle/peBxiio8u99QvKMk6

Thank you in advance for your time and apologize if this is not a proper use of this channel.

@raux
Raula, PhD on behalf of Hideaki, Sebastian, Nicole and Christoph

**Comment by michaelklishin** [comment, marked as answer]
I have filled out the survey on behalf of our team.
